daily_post.py

```diff
@@ -21,25 +21,35 @@
 
 now = datetime.datetime.now()
 
+
 def get_weather():
     resp = requests.get('http://forecast.weather.gov/MapClick.php?lat=47.62&lon=-122.36&unit=0&lg=english&FcstType=text&TextType=1')
     doc = resp.content
     soup_doc = BeautifulSoup(doc, 'html.parser')
-    weather_info = soup_doc.find_all('table')[1]
-    forecast = weather_info.contents[0].text
-    reddit_comment = '* ' + str.replace(forecast, '\n\n',"\n* ")[:-3]
+    weather_info = soup_doc.find_all('table')[1].contents[0].contents[0]
+    reddit_comment = ''
+    if len(weather_info.find_all('div')) > 0:
```

Comment:
Oh boy, this is a pretty big assumption, but we can see if it bites us in the ass at all. Let's just keep an eye on it I suppose.